engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -626,7 +626,7 @@ object ProgramExecutionSupport {
               .getOrElse(expressionValue.getClass)
           ctx.executionService.getLogger.log(
             Level.WARNING,
-            "Execution of visualization [{0}] on value [{1}] of [{2}] failed. {3} | {4}",
+            "Execution of visualization [{0}] on value [{1}] of [{2}] failed. {3} | {4} | {5}",
```

Comment:
We had 6 interpolations here but only 5 of them were being used. I've added the sixth. we can alternatively stop adding the `error` into the interpolation I guess?